dist/uPlot.iife.js

```diff
@@ -4564,8 +4564,13 @@ var uPlot = (function () {
 					if (val != null) {
 						if (ori == 0)
 							x = canOffs[i];
-						else
+						else {
+							if (axis.side === 1 && axis.align === 2)
+								x = can.width;
```

Comment:
I'm guessing 0 to canvas width is not the correct boundaries we want - what are the best offsets?